part1/restaurant.rb

```diff
@@ -0,0 +1,12 @@
+class Restaurant
+  def initialize(menu)
+    @menu = menu
+  end
+
+  def cost(*orders)
```

Comment:
Good, you know how to use it

but pls take a look https://medium.freecodecamp.org/rubys-splat-and-double-splat-operators-ceb753329a78